src/components/photoGallery/PhotoGallery.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState, useEffect } from "react";
+import "./PhotoGallery.scss"
+
+export const PhotoGallery: React.FunctionComponent = () => {
+
+  const [imageUrls, setImageUrls] = useState<string[]>();
+  const img_urls: string[] = [];
+
+        useEffect(() => {
+          const url = `https://api.nasa.gov/planetary/apod?count=50&api_key=DEMO_KEY`;
+          
+          const fetchData = async () => {
+            
+              const response = await fetch(url);
+              const json = await response.json();
+              //json.forEach((item: {url: string}) => img_urls.push(item.url));  
+              setImageUrls(json.map(item => item.url));
+          };
+      
+          fetchData();
+      }, []);
+
+  return (
+      <div className = "image-gallery">
+        {
+          imageUrls === undefined
+          ? <p>Loading....</p>
+          : imageUrls.map(url => <img 
+        className="space-image" src={url} alt = ""/>)}
+      </div>
+  );
+};
```

Comment:
The indenting in this file is off, I'm afraid! Please re-indent this file with 2 spaces per level (you'll need to set VSCode to use 2 spaces in the bottom right of the screen:)
![image](https://user-images.githubusercontent.com/4410524/189181806-b2916762-f215-425c-8558-7192b91544eb.png)